templates/al2/provisioners/install-worker.sh

```diff
@@ -149,7 +149,27 @@ sudo yum install -y runc-${RUNC_VERSION}
 sudo yum versionlock runc-*
 
 # install containerd and lock version
-sudo yum install -y containerd-${CONTAINERD_VERSION}
+# TO-DO: this is a temp way to install binary from s3, need to change once we figure out how to get and store containerd binaries in long term
+if [[ "$CONTAINERD_VERSION" == "1.7.*" && "$INSTALL_CONTAINERD_FROM_S3" == "true" ]]; then
+  CONTAINERD_BINARIES=(
+    containerd
+    containerd-shim
+    containerd-shim-runc-v1
+    containerd-shim-runc-v2
+    ctr
+  )
+  echo "Pulling and installing local containerd binary from s3 bucket"
+  for bianry in "${CONTAINERD_BINARIES[@]}"; do
+    aws s3 cp --region ${BINARY_BUCKET_REGION} s3://${BINARY_BUCKET_NAME}/containerd/${bianry} .
```

Comment:
```suggestion
  for binary in "${CONTAINERD_BINARIES[@]}"; do
    aws s3 cp --region ${BINARY_BUCKET_REGION} s3://${BINARY_BUCKET_NAME}/containerd/${binary} .
```

Reply:
I think we need a version in the S3 key. If the `INSTALL_CONTAINERD_FROM_S3` option is used, we should expect a fully-formed version string in `CONTAINERD_VERSION`.